app/views/overrides/activation_keys/_host_tab_pane.html.erb

```diff
@@ -1,5 +1,6 @@
-
-<%= javascript "katello/hosts/activation_key_edit" %>
+<% content_for(:javascripts) do -%>
+  <%= javascript "katello/hosts/activation_key_edit" %>
+<% end %>
 
```

Comment:
this is redundant, `<%= javacript …` already does `content_for(:javascripts)`: https://github.com/theforeman/foreman/blob/262b5caceab9c3f71d0b80f189cc724389593fb0/app/helpers/layout_helper.rb#L98-L100